src/app/modules/auth-private/sign-up/sign-up.component.ts

```diff
@@ -236,4 +282,58 @@ export class SignUpComponent implements OnInit {
     }
   }
 
+  onInvitationSignup() {
+    this.route.queryParams
+    .subscribe(async (params) =>  {
+      if (!isObjectEmpty(params)) {
+        const {email, first_name, last_name, consultation_id, invitation_token} = params;
+        const isActiveConsultation = await this.checkActiveConsultation(consultation_id);
```

Comment:
@Nishanthh This step needs to be removed now, right?

Reply:
@avikt yes, removed